src/components/UserInfo.js

```diff
@@ -15,4 +16,12 @@ export default class UserInfo {
     this._nameElement.textContent = name;
     this._aboutElement.textContent = about;
   }
+
+  getUserAvatar() {
+    return { userAvatar: this._userAvatar.src };
+  }
+
+  setUserAvatar(avatar) {
```

Comment:
It's great to have separate methods for the avatar, since it's controlled by a different form